app/src/main/java/ch/hikemate/app/ui/map/RunHikeScreen.kt

```diff
@@ -334,11 +342,14 @@ private fun parseLocationUpdate(
   val completionPercentage =
```

Comment:
Warning: This returns a ratio, not a percentage, please rename or change the return

Reply:
Done